bin/cli.js

```diff
@@ -201,8 +200,8 @@ const runBrowser = (config, cb) => {
     cb(1);
   }
 
-  const buildDir = path.resolve('./build');
-  const buildAdapter = path.resolve('./build/adapter.js');
+  const buildDir = path.join(process.env.PWD, 'build');
```

Comment:
I don't think `PWD` works.
> process.env.PWD is the working directory when the process was started

I think the only way to reliably refer to the other directory is `__dirname`.

Reply:
That is exactly why we need `process.env.PWD`. Adapter should have the same parent directory with all modules, libraries, tests etc. which will be loaded in the browser. And this directory should be stated [here](https://github.com/metarhia/metatests-browser-runner/pull/1/files#diff-22d5d2b57c69d2e1ac054c4d0f744ff1R157). FYI, using nearest common parent directory is not an option because karma will try to scan for all files that may be in it possibly loading many files that don't have any relation to package that should be tested resulting in slowdown and erroring if it doesn't have rights to access files.